.github/dependabot.yml

```diff
@@ -5,16 +5,25 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-patch"]
```

Comment:
BTW, patch releases usually responsible for security fixes. Let's look how it will work, but probably we will have more security warnings.